tests/parser/object-literal-method.ts.fixture.ts

```diff
@@ -0,0 +1,5 @@
+const obj = {
+  add: function(a: i32, b: i32): i32 {
```

Comment:
This should be identical to the source file.
https://github.com/AssemblyScript/assemblyscript/pull/1788#discussion_r607950828

The proper way would be to parse to a method declaration, [like here](https://ts-ast-viewer.com/#code/MYewdgzgLgBCBGArGBeGBvAUDGBDAJvgBS4BcMAlgMwBMANDPOdTQJTO0bY4wBOAplACuvMHhgBqRgG5uAX0xzpQA) and modify the compiler to support it.

Reply:
Thanks, "AST Viewer" is really useful.